test/integration/Highlight.e2e.test.js

```diff
@@ -0,0 +1,32 @@
+const highlightAnnotationSelector = '[data-testid^="ba-AnnotationTarget"].ba-HighlightTarget';
```

Comment:
I'm not really a fan of this convention. We've thus far avoided using constants for selectors elsewhere in our tests. Can we simplify the selector to `.ba-HighlightTarget`?